GeoIP2/project.json

```diff
@@ -2,25 +2,28 @@
   "version": "2.7.0-beta2",
   "name": "MaxMind.GeoIP2",
   "description": "MaxMind GeoIP2 Database Reader and Web Service Client",
+
   "packOptions": {
-    "authors": [
-      "MaxMind, Inc."
-    ],
-    "tags": [
-      ""
-    ],
+    "authors": ["Gregory Oschwald", "Mark Fowler"],
+    "tags": ["maxmind", "ip", "geoip", "geoip2", "geolocation", "ip2location", "iptolocation", "maxmind", "ipv4", "ipv6"],
```

Comment:
I just noticed `ip2location` un the tags. Could you remove this? I believe it is trademarked by a competitor.